docker/php/devlocal.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
I'd consider making this file the `CMD` in the php container so that everything just starts up as soon as it starts running